gazebo/rendering/Distortion.cc

```diff
@@ -122,6 +130,11 @@ void Distortion::Load(sdf::ElementPtr _sdf)
   {
     this->dataPtr->compositorName = _sdf->Get<std::string>(compositorName);
   }
+  const std::string legacyMode = "ignition:legacy_mode";
+  if (_sdf->HasElement(legacyMode))
+  {
+    this->dataPtr->legacyMode = _sdf->Get<bool>(legacyMode);
```

Comment:
just to be safe, I would set `this->dataPtr->distortionCrop = false;` here as well

Reply:
@kjeppesen1 ok maybe my comment here is necessary, not "just to be safe"

I think I prefer this approach over 4abe770